packages/showcases/react/stories/components/VtmnLink/VtmnLink.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { VtmnLink } from '@vtmn/react';
+
+import { Meta, Story } from '@storybook/react';
+
+export default {
+  title: 'Components/VtmnLink',
+  component: VtmnLink,
+  argTypes: {
+    href: {
+      type: { name: 'string', required: true },
+      description: 'The href of the link.',
+      defaultValue: '#',
+      control: { type: 'text' },
+    },
+    target: {
+      type: { name: 'string', required: true },
+      description: 'The target of the link.',
+      defaultValue: '_blank',
+      control: { type: 'text' },
+    },
+    size: {
+      type: { name: 'string', required: false },
+      description: 'The size of the link.',
+      defaultValue: 'medium',
+      control: {
+        type: 'select',
+        options: ['small', 'medium', 'large'],
+      },
+    },
+    isStandalone: {
+      type: { name: 'boolean', required: false },
+      description: 'If the component is a standalone or not.',
+      defaultValue: false,
+      control: { type: 'boolean' },
+    },
+    children: {
+      type: { name: 'string', required: false },
```

Comment:
The 'children' represents the text of the link isn't it ? 
If it is, I think it should be required : 'true' because I don't see a link without text.

Reply:
It could be anything. Text, tag contains text, text and icon... ;) 
Anything in our component : 
<VtmnLink>
  <img src="what-i-provide"></img>
</VtmnLink>

Reply:
Yes so that's why I'm ok, children can be required. Because img, text, tag contains text, text and icon are children nodes